proj/src/actions/projectsActions.js

```diff
@@ -1,15 +1,76 @@
 import * as AT from 'actions/types'
 
+export const fetchProjects = () => ({
+  payload: {
+    onSuccess: setProjects,
+    onError: setError
+  },
+  meta: {
+      type: 'database',
+      databaseURL: 'projects',
+      method: 'get'
+  }
+})
+
+export const setProjects = ({projects}) => ({
+  type: AT.SET_PROJECTS,
+  projects
+})
+
+export const setAddedProject = (item) => ({
+  payload: {
+    onSuccess: addProject,
+    onError: setError,
+    item
+  },
+  meta: {
+      type: 'database',
```

Comment:
type: 'firebase' could have been a better name since it tells that this action is destined for firebase middleware. 'database' is a bit generic IMHO